build.psm1

```diff
@@ -529,11 +529,7 @@ function Start-BuildPowerShellNativePackage
 
         [Parameter(Mandatory = $true)]
         [ValidateScript({Test-Path $_ -PathType Leaf})]
-        [string] $macOSZipPath,
-
-        [Parameter(Mandatory = $true)]
-        [ValidateScript({Test-Path $_ -PathType Leaf})]
-        [string] $psrpZipPath
+        [string] $macOSZipPath
     )
```

Comment:
Please let me know if there is anything related to build that I need to clean up in the PowerShell repo. I did a search of `libpsrp` in the scripts and .yml files in the PS repo but didn't find anything.